package.json

```diff
@@ -11,6 +11,7 @@
     "test:coverage": "vitest run --coverage"
   },
   "dependencies": {
+    "@bcrs-shared-components/mixins": "^1.1.16",
```

Comment:
I'm glad this works.

Does Storybook need to depend on all te other shared items here?

Reply:
Not sure, but I don't think so.
All other shared items seem to work completely fine without being added here. Not sure why this one was different.

Reply:
I don't know either so just leave it, then.

Resolving.